src/components/ZclEndpointCard.vue

```diff
@@ -519,8 +519,7 @@ export default {
       get() {
         return (
           this.getDeviceCategory(this.deviceType[0].packageRef) === 'zigbee' &&
```

Comment:
please modify the zigbee word in this condition to DbEnum.helperCategory.zigbee

Reply:
fixed, thank you